tests/conftest.py

```
@@ -0,0 +1,25 @@
+
+_REPO_ROOT = Path(__file__).resolve().parents[1]
+_STYLES = _REPO_ROOT / "styles"
+
+_VALE_INI = {
+    "__root__": {"MinAlertLevel": "suggestion"},
+    "[*.md]": {"BasedOnStyles": "concordat"},
+}
+
+
+@pytest.fixture
+def concordat_vale() -> typ.Generator[Valedate, None, None]:
```

Ruff (0.14.3)

Unnecessary default type arguments

Remove default type arguments

(UP043)